pages/projects.js

```diff
@@ -3,18 +3,36 @@ import {
   SectionTitle,
   Container
 } from '../components/shared-styled'
+import {
+  Project,
+  ProjectsContainer
+} from '../components/projects'
 
 export default function Projects() {
     return (
-      <Container>
+      <>
         <Head>t
           <title>Illini Blockchain</title>
-          <meta name="description" content="Generated by create next app" />
-          <link rel="icon" href="/favicon.ico" />
+          <meta name='description' content='Generated by create next app' />
+          <link rel='icon' href='/favicon.ico' />
```

Comment:
while we're at it, let's change these to have meaningful names and descriptions for that sweet, sweet SEO